src/doc/src/guide/creating-a-new-workspace.md

```diff
@@ -0,0 +1,75 @@
+# Creating a New Workspace
+
+A [workspace][def-workspace] is a collection of one or more packages, called workspace members, that are managed together.
+
+In this chapter we implement the creation of a workspace `new_workspace` from scratch, in which two members `foo-bin` and `bar-lib` are included.
+
+As mentioned in [[workspace] section](../reference/workspaces.md#the-workspace-section), the workspace must have at least one member, either the root package or a virtual manifest. Here we recommend setting the root of the workspace to be a virtual manifest. The reason is as follows:
+
+- The crate namespace at the Cargo level is flat. The tree layout creates another hierarchy and increases the possibility of inconsistencies. 
+- With a flat structure, adding or splitting crates is very easy. 
+- Even larger flattened lists are easier to see at a glance and easier to maintain than smaller tree structures.
```

Comment:
This feels like its blurring the line between a guide-level document and the best practices examples that would hang off the reference.

Seeing this proposed document, I feel like its not pulling enough weight and I wonder if we should first focus in that examples / practices document

Reply:
Perhaps you mean that if add this section it might become a best practice type of article rather than an instructional one. Then I have adjusted the article to make it more suitable as an instructional article for newbies.

Reply:
> Is there anything else that needs to be changed on this, please?


That went in the opposite direction of what I was suggesting.  I'll have to give this more time to think about this to avoid any bias from my own way of wanting to do things.